model/src/main/java/com/javahelp/model/user/Gender.java

```diff
@@ -0,0 +1,8 @@
+package com.javahelp.model.user;
+
+public enum Gender {
+    MALE,
+    FEMALE,
+    OTHERS,
+    UNMENTIONED
```

Comment:
The unmentioned option will be corresponding to the "prefer not to tell" button, right?

Reply:
Yup, and it can be described default value.